src/main/java/core/basesyntax/service/impl/AddingFruit.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.daily.Fruit;
+import core.basesyntax.daily.Storage;
+
+public class AddingFruit implements StockChangeable {
+    private Storage storage;
+
+    @Override
+    public void apply(Fruit fruit) {
+        Storage str = new Storage();
```

Comment:
nope. Accept Storage as a constructor parameter

Reply:
ok. For this I have to make 'add' method (as well as 'remove') as static, right?